engine/Shopware/Plugins/Default/Frontend/Notification/Bootstrap.php

```diff
@@ -109,13 +109,16 @@ public function onPostDispatch(Enlight_Event_EventArgs $args)
 
         if (!empty($notificationProducts)) {
             $sql = 'SELECT `ordernumber` FROM `s_articles_details` WHERE `articleID`=?';
-            $ordernumbers = $this->get(Connection::class)->fetchColumn($sql, [$id]);
+            $ordernumbers = $this->get(Connection::class)->fetchAllAssociative($sql, [$id]);
```

Comment:
```suggestion
            $ordernumbers = $this->get(Connection::class)->fetchAssociative($sql, [$id]);
```

that should give you a list of order numbers directly. so you do not need to iterate again, for picking the order numbers